convert_hf_to_gguf.py

```diff
@@ -454,6 +453,23 @@ def from_model_architecture(cls, arch: str, model_type = ModelType.TEXT) -> type
 
 
 class TextModel(ModelBase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if "text_config" in self.hparams:
+            # move the text_config to the root level
+            self.hparams = {**self.hparams, **self.hparams["text_config"]}
+
+        self.block_count = self.find_hparam(["n_layers", "num_hidden_layers", "n_layer", "num_layers"])
```

Comment:
```suggestion
        self.block_count = self.find_hparam(["n_layers", "num_hidden_layers", "n_layer", "num_layers", "depth"])
```

Qwen VL model use `"depth"`

Reply:
oops I was suppose to edit the code in `VisionModel` instead

Reply:
fixed in https://github.com/ggml-org/llama.cpp/pull/13122/commits/1a0485d52e84c1cc7f73fad140599307e2f1b286